docs/recipe_file.md

```diff
@@ -229,7 +228,7 @@ source:
 ```
 
 Note: `tag` or `rev` may not be available within commit depth range, hence we don't
-allow using `rev` or `tag` and `depth` of them together if not set to `-1`.
+allow using `rev` or the `tag` and `depth` of them together if not set to `-1`.
```

Comment:
I found the original sentence to be a bit confusing but I'm still not 100% sure that this edit makes sense.